<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ui_LineRelAngleOptions</class>
 <widget class="QWidget" name="Ui_LineRelAngleOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>221</width>
    <height>24</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>22</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>280</width>
    <height>32767</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Line Relative Angle Options</string>
  </property>
  <layout class="QHBoxLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QLabel" name="lAngle">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Angle:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leAngle">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Line angle</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lLength">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Length:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leLength">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Length of line</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>updateAngle(QString)</slot>
  <slot>updateLength(QString)</slot>
 </slots>
</ui>
